File: program/convert-string-to-alternatingcase/ConvertStringToAlternatingcase.java
import java.util.Scanner;
public class alternatingCase {
   public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      System.out.println("Enter a string :");
      String str = sc.nextLine();
      str = str.toLowerCase();
      char[] ch = str.toCharArray();
      for(int i=0; i<ch.length; i=i+2){
         ch[i] = Character.toUpperCase(ch[i]);
      }
      System.out.println(new String(ch));
   }
}

File: program/program/convert-string-to-alternatingcase/ConvertStringToAlternatingcase.kt
fun alternatingCase(input: String): String {
    return input.mapIndexed { index, char ->
        if (index % 2 == 0) {
            char.toLowerCase()
        } else {
            char.toUpperCase()
        }
    }.joinToString("")
}

fun main() {
    println("Enter a string: ")
    val inputString = readLine() ?: ""
    val convertedString = alternatingCase(inputString)
    println("Alternating Case String: $convertedString")
}
